api/mapping/metadatacatalogue/event/MetadataSearchResultEvent.md

```diff
@@ -0,0 +1,64 @@
+# MetadataSearchResultEvent [rpc]
+
+Notifies that a metadata search has been performed and the result is accessible through the event.
+
+## Description
+
+Used to notify that the ``MetadataSearchRequest`` has received a reply from server side search.
+
+## Parameters
+
+(* means the parameter is required)
+
+<table class="table">
+<tr>
+  <th> Name</th><th> Type</th><th> Description</th><th> Default value</th>
+</tr>
+<tr>
+  <td> \* results</td><td> Object </td><td> metadata search results object</td><td> </td>
+</tr>
+<tr>
+  <td> \* error</td><td> Boolean </td><td> whether an error occurred during the search </td><td> </td>
```

Comment:
The "regular" search uses success instead of error: https://github.com/oskariorg/oskari-frontend/blob/master/api/framework/search/event/SearchResultEvent.md

Change this to match for consistency.